### PR TITLE
Refactoring @borrowbot twitter bot

### PR DESCRIPTION
**Pull request by mekarpeles**
1. Simplifying main runner flow
2. Centralizing tweet content
3. Moving models isbn + book APIs to separate file
4. Adding test cases

### CODE REVIEW

**Comment by mekarpeles** [comment]
@tabshaikh if you'd like to help review

File: twitter-borrowbot/services.py [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+                ("fl[]", "lending___is_printdisabled"),
+                ("rows", "50"),
+                ("page", "1"),
+                ("output", "json")
+            ]
+            matches = requests.get(url, params=params).json()
+            if matches and matches["response"]["docs"]:
+                books = matches["response"]["docs"]
+                return next(book for book in books if book.get("openlibrary_work"))
+        except:
```
Comment (cclauss) [inline twitter-borrowbot/services.py:101]:
```suggestion
        except Exception:
```

File: twitter-borrowbot/twitterbot.py [hunk 2]
```diff
@@ -1,153 +1,140 @@
+#-*- encoding: utf-8 -*-
```
Comment (cclauss) [inline twitter-borrowbot/twitterbot.py:1]: (Only required on Python 2.  Python 3 files are UTF-8 by default)

File: twitter-borrowbot/twitterbot.py [hunk 3]
```diff
@@ -1,153 +1,140 @@
...
 import re
 import time
 import requests
 import tweepy
-import isbnlib
 
+from services import InternetArchive, ISBNFinder
 from dotenv import load_dotenv
+
+ACTIONS = ['read', 'borrow', 'preview']
```
Comment (cclauss) [inline twitter-borrowbot/twitterbot.py:11]:
```suggestion
ACTIONS = ('read', 'borrow', 'preview')
```

**Review by cclauss — APPROVED** [review]
LGTM... Non-crucial suggestions.

**Pull request merged**